src/content/guides/creating-token-collections.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Creating Token Collections
+description: Create token collections, define redemption conditions, and mint tokens for the collections.
```

Comment:
This is quite literal - should be more sale-sy to encourage people to click the link. A shorter version of what you've got under `What is a Digital Token`

Reply:
Will do, I didn't think this was actually surfaced on the page, where do you see it?

Reply:
@seliwe It gets added to the page metadata.

You'll see this in search engine results and in link previews when the link is shared on social sites (LinkedIn, Facebook, Twitter, Outlook) etc